frontend/vitest.setup.ts

```
@@ -113,3 +97,10 @@ if (typeof Element !== 'undefined') Element.prototype.animate = vi.fn().mockImpl
   };
   return mock as unknown as Animation;
 });
+
+// Reset storage and DOM between every test (required for isolate: false)
+beforeEach(() => {
+  Object.keys(localStorageStore).forEach(key => delete localStorageStore[key]);
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)